frontend/components/Course/CourseWrapper.tsx

```diff
@@ -65,47 +69,51 @@ const CourseWrapper = ({ render, ...props }: CourseProps) => {
                             </Segment>
                         </Grid.Column>
 
-                        <Grid.Column>
-                            <Segment basic>
-                                <div
-                                    style={{
-                                        float: "right",
-                                        paddingTop: "0.71rem",
-                                    }}
-                                >
+                        {notificationUI && (
+                            <Grid.Column>
+                                <Segment basic>
                                     <div
                                         style={{
-                                            display: "inline",
-                                            position: "relative",
-                                            top: "0.14rem",
-                                            fontSize: "1.29rem",
-                                            fontFamily: "Lato",
-                                            color: "#666666",
+                                            float: "right",
+                                            paddingTop: "0.71rem",
                                         }}
                                     >
-                                        Notifications are{" "}
-                                        {notifs ? "ON" : "OFF"}
+                                        <div
```

Comment:
I think we talked about this, and on second thought perhaps we should add a small info indicator here that displays on hover and lets the user know that they should enable notification permissions if possible. 

I think we can even use https://developer.mozilla.org/en-US/docs/Web/API/Notification/permission to tell whether we have permissions, but I'm actually not super sure if there's a great way to re-render the component when it changes, so it *might* be stale so we don't have to do this. If you know how to do this that'll be awesome though